src/test/java/org/example/UserServiceTest.java

```diff
@@ -0,0 +1,121 @@
+package org.example;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class UserServiceTest {
+
+    private final UserService userService = new UserService();
+
+    private final UserGenerator userGenerator = new UserGenerator();
+
+    private final List<Client> clientList = userGenerator.getClients();
+
+    @Test
+    void should_getSumByAge() {
+        //given
+        String name = "bob";
+
+        //when
+        int sum = userService.getSumAgeByName(clientList, name);
+
+        //then
+        int expectedSum = 820;
+        assertEquals(expectedSum, sum);
```

Comment:
what library do we use to assert values ? 
https://www.notion.so/f48a49a6e20845c2a78c8f304a84f5f2?v=a327579a7f664e66b4364637bc0b177f&p=72de573740bb406784352f65532492ba&pm=c